src/components/Card/Chart/TodayPostCountCard.jsx

```diff
@@ -0,0 +1,33 @@
+import asyncGetTodayPostCount from "../../../api/keyword/asyncGetTodayPostCount";
+import { useQuery } from "@tanstack/react-query";
+import PropTypes from "prop-types";
+
+const TodayPostCountCard = ({ keywordId }) => {
+  const { data: chartData } = useQuery({
+    queryKey: ["todayPostCount", keywordId],
+    queryFn: () => asyncGetTodayPostCount(keywordId),
+  });
+
+  if (typeof chartData === "undefined") {
+    return <div className="mt-100 w-1/3 p-10">조회된 게시물이 없습니다 🥲</div>;
+  }
+
+  return (
+    <article className="w-1/2 h-full p-10 border-2 rounded-md">
+      <span>오늘의 게시물</span>
+      <p className="flex justify-center mt-20">
+        어제대비 게시물 수 {Math.abs(chartData.diffPostCount)}개{" "}
+        {chartData.diffPostCount >= 0 ? "증가" : "감소"}
+      </p>
```

Comment:
```suggestion
      <p className="flex justify-center mt-20">
        {Number(chartData.diffPostCount) === 0 ? "어제와 게시물 수가 동일해요." : (
          <>
            어제 보다 {Math.abs(chartData.diffPostCount)}개
            {Number(chartData.diffPostCount) > 0 ? "증가" : "감소"}
          </>
        )}
      </p>
```

1. 어제 대비 (1) 동일, (2) 증가, (3) 감소 케이스를 모두 다뤄요.
  - 현재 증가, 감소 케이스 밖에 없어서 어제와 동일한 경우 어색하게 나올 수 있어서 3가지 케이스 모두 대응이 필요하겠습니다!
  - 아래 테스트 해본 UI 첨부했어요.
2. 아래처럼 `diffPostCount`가 문자열로 오다보니, 비교 연산자를 정확하게 사용하기 위해 `Number()`로 변환해주었습니다.

```js
// postController.today
return res.status(200).json({
   // ...   
   diffPostCount: (todayPostCount - yesterdayPostCount).toString(),
    });
```
##### 참고) 어제와 게시물 수 동일할 때
<img width="300" alt="Screenshot 2024-11-13 at 6 19 30 PM" src="https://github.com/user-attachments/assets/07b5dda7-a726-447e-b564-daee2474abfe">

Reply:
게시물 수가 동일한 경우는 생각하지 못했는데 감사합니다! 
1. 게시물 수가 동일한 경우 -> `어제와 게시물 수가 동일합니다.`
2. 게시물 수가 어제보다 많은 경우 -> `어제 대비 게시물 수가 $$개 증가 `
3. 게시물 수가 어제보다 적은 경우 -> `어제 대비 게시물 수가 $$개 감소`

3개의 경우에 대한 문구는 이렇게 수정했습니다.